site/content/docs/5.0/migration.md

```diff
@@ -12,7 +12,7 @@ toc: true
 ### Sass
 
 #### Utilities
-
+- Removed `.rounded-sm` and `rounded-lg` and introduced `.rounded-0` to `.rounded-3`.
```

Comment:
This should be under the v5.0.0-alpha3 migration guide, not the beta2 one. https://getbootstrap.com/docs/5.0/migration/#v500-alpha3

Can you please move it, and add a link to the PR? https://github.com/twbs/bootstrap/pull/31687

Reply:
Sure thing !